e2e/tcproute_test.go

```diff
@@ -441,3 +443,288 @@ func defaultNetworkURL() string {
 func defaultSubnetworkURL() string {
 	return cloud.NewSubnetworksResourceID(testFlags.project, region, "default").SelfLink(meta.VersionGA)
 }
+
+func createManyHealthchecks(graphBuilder *rgraph.Builder, hcNum int, name string) ([]*cloud.ResourceID, error) {
```

Comment:
I don't think this function is needed can you merge it with `createManyBackendServicesWithHC`?

```
func createManyBackendServicesWithHC(graphBuilder *rgraph.Builder, bsNum int, name string) ([]*cloud.ResourceID, error){
for i := 0; i < bsNum; i++ {
hcName := "hc-"+name+"-"+strconv.Itoa(i)
hc, err := buildHealthCheck(graphBuilder, hcName, 15)
if err != nil {
		errors.Join(fmt.Errorf("buildHealthCheck(%s) err: %w",e), hcName, err))
                continue
}
bsName := "bs-"+name+"-"+strconv.Itoa(i)
bs, err := buildBackendServiceWithLBScheme(graphBuilder, bsName, hc, "INTERNAL_SELF_MANAGED")
if err != nil {		
		errors.Join(fmt.Errorf("buildBackendServiceWithLBScheme(%s) err: %w",e), bsName, err))
		bss = append(bss, bs)
}
}
}
```

Reply:
Done.